src/Microsoft.ML.Data/Properties/AssemblyInfo.cs

```diff
@@ -26,6 +26,7 @@
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.HalLearners" + PublicKey.Value)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.KMeansClustering" + PublicKey.Value)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.LightGBM" + PublicKey.Value)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.LightGBM.StaticPipe" + PublicKey.Value)]
```

Comment:
Will we still need this after @tomfinley's comments are addressed?  (Basically to keep a `public string WeightColumn` property)